library/testing/TestSDKCamera.cxx

```diff
@@ -0,0 +1,230 @@
+#include <camera.h>
+#include <engine.h>
+#include <loader.h>
+#include <window.h>
+
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+
+bool compareDouble(double a, double b)
+{
+  return std::fabs(a - b) < 128 * std::numeric_limits<double>::epsilon();
+}
+
+bool compareVec(f3d::camera::vector3_t vec1, f3d::camera::vector3_t vec2)
```

Comment:
I'm nitpicking here so feel free to ignore. A better way to compare two vectors is to check the length is the same and that the angle between them is zero.

Reply:
it is indeed a better geometrical approach, but does not seem necessary in this context.